tests/ui/test_pvc_ui.py

```diff
@@ -130,11 +145,63 @@ def test_create_delete_pvc(
         )
         logger.info("PVC Details Verified via UI..!!")
 
+        logger.info("Creating Pod")
+
+        if sc_type in (
+            constants.DEFAULT_STORAGECLASS_RBD_THICK,
+            constants.DEFAULT_STORAGECLASS_RBD,
+        ):
+            interface_type = constants.CEPHBLOCKPOOL
+        else:
+            interface_type = constants.CEPHFILESYSTEM
+        new_pod = helpers.create_pod(
+            interface_type=interface_type,
+            pvc_name=pvc_name,
+            namespace=project_name,
+            raw_block_pv=vol_mode == constants.VOLUME_MODE_BLOCK,
+        )
+
+        logger.info(f"Waiting for Pod: state= {constants.STATUS_RUNNING}")
+        wait_for_resource_state(resource=new_pod, state=constants.STATUS_RUNNING)
+
+        teardown_factory(new_pod)
+
+        logger.info("Pvc Resizing")
+        new_size = int(pvc_size) + 1
+        pvc_ui_obj.pvc_resize_ui(
+            pvc_name=pvc_name, new_size=new_size, project_name=project_name
+        )
+
+        logger.info("Verifying New Capacity after Pvc Resize")
+        assert new_size > int(
+            pvc_size
+        ), f"New size of the PVC cannot be less than existing size: new size is {new_size})"
+
+        baseui_obj = PageNavigator(setup_ui)
```

Comment:
@am-agrawa why do you use `PageNavigator` class here?

Reply:
removed